src/Controller/UsersController.php

```diff
@@ -437,24 +437,69 @@ public function deleteUser(Request $request, PDO $db)
      * @throws Exception
      */
     public function setTrusted(Request $request, PDO $db)
+    {
+        $this->setAdminOnlyBooleanField($request, $db, 'trusted');
+    }
+
+    /**
+     * Allow users to be set as admin
+     *
+     * @param $request Request
```

Comment:
These 2 `@param` annotations don't really add anything.

This comment can probably be removed completely.

Reply:
Storm complains about uncaught, un-annotated exceptions if I drop the docblocks. Just pushed a commit that suppresses these complaints, which allows for dropping docblocks without highlighting the sole method call here.